### PR TITLE
Fix formatting in Packaging guide's Compiler wrappers section

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
CI failure shouldn't be related to the PR, since it only touches documentation

File: lib/spack/docs/packaging_guide.rst [hunk 1]
```diff
@@ -6375,4 +6368,4 @@ To achieve backward compatibility with the single-class format Spack creates in
 
 Overall the role of the adapter is to route access to attributes of methods first through the ``*Package``
 hierarchy, and then back to the base class builder. This is schematically shown in the diagram above, where
-the adapter role is to "emulate" a method resolution order like the one represented by the red arrows.
\ No newline at end of file
+the adapter role is to "emulate" a method resolution order like the one represented by the red arrows.
```
Comment (alalazo) [inline lib/spack/docs/packaging_guide.rst:6384]: Can you undo the change to this line? It seems unrelated (and invisible :slightly_smiling_face: )

**Review by alalazo — CHANGES_REQUESTED** [review]

File: lib/spack/docs/packaging_guide.rst (hunk 1 above)
Comment (iarspider) [reply lib/spack/docs/packaging_guide.rst:6384]: And I made it worse :(
